src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -412,6 +412,17 @@ public JSONObject getTranslationJobs(List<Integer> ids, JobStatus status, Intege
      */
     public JSONObject reviseTranslationJobs(HashMap<Integer, String> jobs) throws GengoException
     {
+        return reviseTranslationJobs(jobs, null);
```

Comment:
This method doesn't take `comment`, so I think you need to update method comment.